.github/workflows/combined_format_lint_test_mypy.yml

```
@@ -2,9 +2,9 @@ name: Format, Lint, Test, Coverage, and Type Checking
 
 on:
   push:
-    branches: [ dev ]
+    branches: [ dev, dev-msf ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)